CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- pre-commit: update configuration
```

Comment:
No need to update the changelog: developers don't care about pre-commit.

Reply:
Oh, hence the no-changelog label, you're right

Reply:
I force-pushed the branch after rebasing & dropping the commit